io.github.ja2_stracciatella.JA2-Stracciatella.yaml

```diff
@@ -0,0 +1,141 @@
+app-id: io.github.ja2_stracciatella.JA2-Stracciatella
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.rust-stable"
+command: ja2-launcher-wrapper
+rename-desktop-file: ja2-stracciatella.desktop
+rename-icon: ja2-stracciatella
+finish-args:
+  - "--share=network"
+  - "--socket=x11"
+  - "--socket=wayland"
```

Comment:
Does it support native wayland? If so, this should be fallback-x11.

Reply:
I'd really like to keep wayland enabled by default.  On xwayland, SDL doesn't respect the display scale (at least it doesn't for me, perhaps I'm just missing a config knob here?), and game assets from the 90s don't really work on a 4k screen 😬 

Now, giving that those are pretty common for gaming these days, xwayland is really poor default in my opinion; the game's almost unplayable on 4k unscaled.

Reply:
it either `x11` or `fallback-x11 + wayland`.

Reply:
That's unfortunate.  The launcher won't work with `fallback-x11` under wayland, and the game is so small as to be unplayable on a 4k screen under wayland.

So I'm either left with a broken launcher or a (semi) broken game?

Reply:
And I mean, I'd not be the only one.  Steam has wayland + x11, signal has it, VSCodium too, and a whole bunch of other things as well 🤷🏿

Reply:
Two wrongs don't make a right. 

Now if one program require X11 but the other can work with wayland, then please use the comment feature of yaml to describe it, because it's not obvious.

I can speak for a few that do it:
- Inkscape: it has extensions that use obsolete toolkits like Tk that only now X11 while the app is wayland.
- Same for Scribus.

VSCodium should not have wayland because it's electron, and electron is still no Wayland compatible out of the box.

Now maybe the launcher should be fixed to use wayland... but that's out of scope here.